lib/Mir.ml

```diff
@@ -50,11 +54,12 @@ type unsizedtype = Ast.unsizedtype [@@deriving sexp, hash]
 type constraint_check =
   {ccfunname: string; ccvid: string; cctype: sizedtype; ccargs: expr list}
 
+and formal_params = (adtype * string * unsizedtype) list
```

Comment:
postfix operators aren't a thing in any of our potential target languages; someone once convinced me that binary operators in C++ are more efficient than function calls (but I'm inclined to just switch back to treating them like fn calls for now), prefix operators idk

Reply:
Could you maybe discuss the binary operator question with @bob-carpenter ? I think he'd be the most likely person to know the best course of action here, right? I don't know what's best myself. When in doubt, I'd be inclined to do what the old code generator does, myself.

Reply:
Ended up talking with Matthijs a lot today and decided to just go with trying to keep things close to existing code gen until we release. BinOp could have gone either way, but the math library doesn't support calling `add(int, int)` so rather than figuring out a way around it, it seems easier to just go back to BinOp. I want to also make tickets or otherwise keep track of opportunities for future simplification...